### ISSUE TITLE
Mobileclient.search(): no album_hits entries

### ISSUE THREAD

**Issue by vinyldarkscratch** [simon-weber/gmusicapi#622]
Tried searching for two different albums with no results for `album_hits` (or anything other than `video_hits` for that matter).  Anyone else having this issue or know of a fix?  If it helps, these are the steps I'm taking:

```py
album_artist = "glitch_d"
album_name = "Arcade EP"

import gmusicapi.Mobileclient
googleplay = gmusicapi.Mobileclient()
googleplay.login(EMAIL, PASSWORD, gmusicapi.Mobileclient.FROM_MAC_ADDRESS)

googleplay_data = googleplay.search('%s %s' %(album_artist, album_name))

print(googleplay_data['album_hits'])
```

**Comment by thebigmunch** [comment]
The fix is already merged but hasn't been released yet.

**Comment by vinyldarkscratch** [comment]
Ah, sweet!  Looking forward to when the fix is released -- we're setting it up as a part of an automatic generation of [these links](https://www.gooborg.com/records/release/glitch-d-arcade-ep).  😛

Thanks for the reply!

**Comment by simon-weber** [comment]
Ok, this is out in 11.0.4.

**Comment by vinyldarkscratch** [comment]
Just got a chance to check it out, and sadly I'm still experiencing issues, but I figured out what's going on.  I tried a bunch more searches for some big pop songs (like "Anaconda", "Taylor Swift", "Miley Cyrus Wrecking Ball") as a little test, rather than just my record label's releases, but no luck.  (I even went in and forced the search to only search for albums and songs.)  I just went to Google Play Music (rather than the Google Play Store) to search for my songs, and in there, only came up with video results.  Out of curiosity, I signed up for the free 30 day trial, and boom, album and track results popping up in both the website and the library.

So, it turns out that it's the limitations of a free Google Play account that was causing it the entire time, rather than anything wrong with the library.  -_-
